resolver/resolver.go

```diff
@@ -74,23 +74,6 @@ func (resolver *Resolver) QueryBlocks(ctx context.Context, id interface{}, param
 }
 
 func (resolver *Resolver) SubmitTx(ctx context.Context, id interface{}, params *jsonrpc.ParamsSubmitTx, req *http.Request) jsonrpc.Response {
-	// When a v0 burn tx gets submitted via RPC, we have to wait for the watcher to detect it before submitting
```

Comment:
Have we tested burn submission via ren-js?

Reply:
No. Would be good if you could help test with ren-js